### PR TITLE
i18n(es): Updated `contribute.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `contribute.mdx` based on these commits 99fe714562b35720768a9e8d93185f2672b8bb32, 762eceb4e8b6d8f5382f0df788d28fa2f778718b & 38f70780398b3a59697d0be8a4eff013a234e8df

### CODE REVIEW

File: src/content/docs/es/contribute.mdx [hunk 1]
```diff
@@ -10,19 +10,39 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
...
 
-Cada PR, especialmente las PRs de traducción, ¡necesita evaluadores! Revisar la PR y dejar comentarios, sugerencias o aprobarla con un "LGTM!" ("¡Me parece bien!") es una gran manera de comenzar en Team Docs y aprender más sobre Astro.
+¡También tenemos una comunidad muy activa de [Discord](https://astro.build/chat)! Nosotros valoramos las contribuciones de aquellos que dan la bienvenida a nuevos miembros, responden preguntas de soporte y comparten lo que han construido con y para Astro. Más allá de las contribuciones tradicionales de GitHub, Astro reconoce y apoya a los miembros de la comunidad que participan con nuestra comunidad, comparten Astro en publicaciones de blog, videos y charlas de conferencias, y ayudan a mantener la salud de nuestra comunidad.
 
-## Guías de contribuyentes
+## Contribuyendo a la documentación
 
-¡Tenemos muchas guías con información sobre cómo comenzar!
+¡Tenemos muchas guías con información sobre cómo comenzar a contribuir con Astro Docs!
```
Comment (Waxer59) [inline src/content/docs/es/contribute.mdx:46]: te falta esta linea del commit [`38f7078`](https://github.com/withastro/docs/commit/38f70780398b3a59697d0be8a4eff013a234e8df)
```suggestion
> ¿Quieres contribuir al código base de Astro? Echa un vistazo a la [guía de contribución de Astro](https://github.com/withastro/astro/blob/main/CONTRIBUTING.md).
```

File: src/content/docs/es/contribute.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Contribuye a la documentación
-description: Una guía para contribuir a la documentación de Astro.
+title: Contribuye a Astro
+description: Cómo involucrarse y contribuir con Astro.
```
Comment (Waxer59) [inline src/content/docs/es/contribute.mdx:3]:
```suggestion
description: Cómo involucrarse y contribuir a Astro.
```

File: src/content/docs/es/contribute.mdx [hunk 3]
```diff
@@ -10,19 +10,39 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
...
 ## Maneras de contribuir
 
-¡Tenemos varias guías con información sobre cómo comenzar a contribuir a la documentación de Astro!
+## Repositorios del proyecto
+
+¡Hay muchas maneras de contribuir al proyecto Astro! Cada repositorio de Astro tiene un README con un enlace a un archivo `CONTRIBUTING.md` en la raíz del proyecto.
+
+Visita el [perfil de GitHub de Astro](https://github.com/withastro) para encontrar los repositorios de:
+
+- La [base de código principal de Astro](https://github.com/withastro/astro), incluyendo integraciones oficiales y plantillas de inicio.
```
Comment (Waxer59) [inline src/content/docs/es/contribute.mdx:23]:
```suggestion
- El [código base principal de Astro](https://github.com/withastro/astro), incluyendo integraciones oficiales y plantillas de inicio.
```

File: src/content/docs/es/contribute.mdx [hunk 4]
```diff
@@ -10,19 +10,39 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
...
+
+- ¡[Astro Docs](https://github.com/withastro/docs), un sitio web completo de Astro! Contribuye no solo con contenido escrito, sino también con código Astro que aborde las preocupaciones de a11y, CSS, UI y UX. También ponemos nuestra documentación a disposición en varios idiomas, por lo que necesitamos ayuda para traducir todo el sitio.
```
Comment (Waxer59) [inline src/content/docs/es/contribute.mdx:25]:
```suggestion
- ¡[Astro Docs](https://github.com/withastro/docs), un sitio web completo de Astro! Contribuye no solo con contenido escrito, sino también con código Astro que aborde las preocupaciones de a11y, CSS, UI y UX. También ponemos nuestra documentación disponible en varios idiomas, por lo que necesitamos ayuda para traducir todo el sitio.
```

File: src/content/docs/es/contribute.mdx [hunk 5]
```diff
@@ -10,19 +10,39 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
...
+
+### Tipos de contribuciones
 
-El sitio web de la documentación de Astro es ... ¡un sitio web de Astro! Mantenerlo requiere no solo contenido escrito, sino también mantener el código de Astro y abordar preocupaciones de a11y, CSS, UI y UX. También hacemos que nuestra documentación esté disponible en varios idiomas, por lo que necesitamos ayuda para traducir todo el sitio.
+Adicionalmente a contribuir con tu propio código o contenido, también puedes hacer una gran contribución involucrándote dejando comentarios de revisión en PRs, agregando ideas en las discusiones y problemas de GitHub existentes, ¡y participando en nuestras tareas de mantenimiento de problemas "Fijados"!
 
-¡También puedes hacer una contribución enorme al involucrarte dejando comentarios de revisión en PR, agregando ideas en problemas y discusiones existentes de GitHub y participando en nuestras tareas de mantenimiento de problemas "Fijados"!
+Cada PR, especialmente los PR de traducción, ¡necesita revisores! Revisar PR y dejar comentarios, sugerencias o un "LGTM!" ("¡Se ve bien para mí!") es una excelente manera de comenzar en cualquier repositorio y aprender más sobre Astro.
```
Comment (Waxer59) [inline src/content/docs/es/contribute.mdx:39]:
```suggestion
Cada PR, especialmente las PR de traducción, ¡necesita revisores! Revisar PRs y dejar comentarios, sugerencias o un "LGTM!" ("¡Se ve bien para mí!") es una excelente manera de comenzar en cualquier repositorio y aprender más sobre Astro.
```

File: src/content/docs/es/contribute.mdx [hunk 6]
```diff
@@ -10,19 +10,39 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
...
 
-Cada PR, especialmente las PRs de traducción, ¡necesita evaluadores! Revisar la PR y dejar comentarios, sugerencias o aprobarla con un "LGTM!" ("¡Me parece bien!") es una gran manera de comenzar en Team Docs y aprender más sobre Astro.
+¡También tenemos una comunidad muy activa de [Discord](https://astro.build/chat)! Nosotros valoramos las contribuciones de aquellos que dan la bienvenida a nuevos miembros, responden preguntas de soporte y comparten lo que han construido con y para Astro. Más allá de las contribuciones tradicionales de GitHub, Astro reconoce y apoya a los miembros de la comunidad que participan con nuestra comunidad, comparten Astro en publicaciones de blog, videos y charlas de conferencias, y ayudan a mantener la salud de nuestra comunidad.
```
Comment (Waxer59) [inline src/content/docs/es/contribute.mdx:41]:
```suggestion
¡También tenemos una comunidad muy activa de [Discord](https://astro.build/chat)! Valoramos las contribuciones de aquellos que dan la bienvenida a nuevos miembros, responden preguntas de soporte y comparten lo que han construido con y para Astro. Más allá de las contribuciones tradicionales de GitHub, Astro reconoce y apoya a los miembros de la comunidad que participan con nuestra comunidad, comparten Astro en publicaciones de blog, videos y charlas de conferencias, y ayudan a mantener la salud de nuestra comunidad.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Review by Waxer59 — COMMENTED** [review]
Te falta eliminar una linea duplicada y listo

**Comment by dreyfus92** [comment]
Listo, gracias @Waxer59  🚀

**Review by Waxer59 — APPROVED** [review]
LGTM ✔️

File: src/content/docs/es/contribute.mdx [hunk 7]
```diff
@@ -10,25 +10,47 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
...
+## Contribuyendo a la documentación
+
+¡Tenemos muchas guías con información sobre cómo comenzar a contribuir con Astro Docs!
 
 - [Guía principal para contribuir](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - ¡Por favor consulta esto primero!
 - [Guía de Redacción](https://github.com/withastro/docs/blob/main/WRITING.md) - Consejos para escribir en el estilo de la documentación de Astro.
 - [Guía de Traducción](https://github.com/withastro/docs/blob/main/TRANSLATING.md)  - Información sobre cómo ayudar a traducir la documentación de Astro a tu idioma.
 - [Guía de Recetas](https://github.com/withastro/docs/blob/main/RECIPES.md) - Aprende sobre cómo escribir y contribuir con una receta.
 
+> ¿Quieres constribuir al código de Astro? Revisa la [guía de contribución de Astro](https://github.com/withastro/astro/blob/main/CONTRIBUTING.md).
```
Comment (Yan-Thomas) [inline src/content/docs/es/contribute.mdx:52]:
```suggestion

```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Hope you don't mind I removed the blockquote we just removed from main so y'all don't have to update the page two times in the same week!

**Pull request merged**